### ISSUE TITLE
Add Manuals to the Downloads Menu

### ISSUE THREAD

**Issue by rkrug** [OSGeo/grass-website#196]
Tne Manuals (pdf) are mainly for downloading, so I would add a section to the Download page and menu with "Manuals".

**Comment by veroandreo** [comment]
IMHO they are more about *learning* how to use the modules. Do users normally download them?

**Comment by neteler** [comment]
In the old site we had zip-downloads listed: https://grass.osgeo.org/documentation/manuals/

While I would avoid another entry in the download dropdown menu it might be harmless to add it here (unless it is not easily possible): https://staging.grass.osgeo.org/download/   - no strong opinion though..

**Comment by mlennert** [comment]
Am 3. Juli 2020 16:57:32 MESZ schrieb Veronica Andreo <notifications@github.com>:
>IMHO they are more about *learning* how to use the modules. Do users
>normally download them?

IIRC, the request to be able to download man pages came from one of our African students in situations with fluctuating internet connection.

I don't know how many people actually use the option, but there was a demand at one point in time.

**Comment by ParamSingh24** [comment]
I agree that adding a "Manuals" section to the Downloads page and menu would be useful. While manuals are primarily for learning, many users also prefer to download PDFs for offline access, easier reference, and annotation.

For example, Blender provides a downloadable manual along with their online documentation, allowing users to choose how they access the content. Similarly, Ubuntu offers official PDF guides for offline reading.

Adding this section would improve accessibility for users who may not always have an internet connection or who prefer a structured, downloadable format.

**Comment by wenzeslaus** [comment]
Note that the documentation for the main tools is generally included with the installation. The download has addons and Python in addition to it, but that can be adjusted. So, the question is not whether to have it available offline, but only about additional ways of obtaining it.
